tests/Microsoft.Bot.Builder.Expressions.Tests/ExpressionEngineTests.cs

```diff
@@ -373,10 +376,29 @@ public class ExpressionEngineTests
             Test("getFutureTime(1,'Month','MM-dd-yy')", DateTime.Now.AddMonths(1).ToString("MM-dd-yy")),
             Test("getFutureTime(1,'Week','MM-dd-yy')", DateTime.Now.AddDays(7).ToString("MM-dd-yy")),
             Test("getFutureTime(1,'Day','MM-dd-yy')", DateTime.Now.AddDays(1).ToString("MM-dd-yy")),
-           
-            # endregion
+            Test("convertFromUTC('2018-01-01T08:00:00.0000000Z', 'Pacific Standard Time', 'D')", "Monday, January 1, 2018"),
+            Test("convertFromUTC('2018-01-01T08:00:00.0000000Z', 'America/Los_Angeles', 'D')", "Monday, January 1, 2018"),
```

Comment:
>2018-01-01T08:00:00.0000000Z [](start = 34, length = 28)

This test can not verify the  function. You should convert from 2018-01-01 to 2018-01-02 to verify it